packages/web-api/src/response/index.ts

```diff
@@ -1,4 +1,4 @@
-export { AdminAnalyticsGetFileResponse } from './AdminAnalyticsGetFileResponse';
+export { AdminAnalyticsGetFileResponse, MemberDetails, PublicChannelDetails, PublicChannelMetadataDetails } from './AdminAnalyticsGetFileResponse';
```

Comment:
TODO: This line needs to be manually added in the code generator script (the Ruby script)